### PR TITLE
Screwsolder

### PR DESCRIPTION
**Pull request by Kurfursten**
🆑 
* rscadd: Gives the chief engineer a screwdriver soldering tool. It holds 60% more acid, completes work in 50% of the time, and eliminates the need to carry a separate screwdriver.

### CODE REVIEW

**Comment by jwhitak** [comment]
Does it have a cool icon?

**Comment by Kurfursten** [comment]
Just a color swap

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -843,6 +844,37 @@
 	reagents.add_reagent(SACID, 50)
 	update_icon()
 
+/obj/item/tool/solder/screw
+	name = "screwsolder"
+	desc = "An advanced solder with a screwdriver head. Use in hand to swap to and from the screwhead."
```
Comment (kernikov) [inline code/game/objects/items/weapons/tools.dm]: "solder" refers to the alloy or metal used, the actual heating is done by a soldering iron or gun.

**Comment by d3athrow** [comment]
Cool but why though

**Comment by kernikov** [comment]
because he's the CE, why not, and the RD still mogs him with the infinite fuel holo welding tool

**Comment by pedr69** [comment]
>RD mogs him with his switchtool

until he clicks on a wheel of cheese with it

**Comment by Eneocho** [comment]
> > RD mogs him with his switchtool
> 
> until he clicks on a wheel of cheese with it

You'd be surprised. Only a few tools can be taken out of the holo that way. If I remember correctly, you can take out the UV, the wrench and the shield. Not sure about the other tools (any sharp tool just cuts the thing).

**Comment by Kurfursten** [comment]
> Cool but why though

CE belt has a lot of competition for slots
I promised a faster CE solder tool back earlier when dorain wanted to make solders instant
I like double tools
Head benefits good, encourage people to play head

**Comment by Eneocho** [comment]
Head benefits good but solders should still be faster as a baseline, 4 seconds to fix an emagged airlock circuitboard is too long.

**Comment by DorainGrey** [comment]
> Head benefits good but solders should still be faster as a baseline, 4 seconds to fix an emagged airlock circuitboard is too long.

Honestly it's fine now, with the instant screwing. It means you can fix an airlock in less than 10 seconds if you are quick with the tool changes. CE specific tools are appreciated though so it's great to see this.

**Comment by ConfusedCactus** [comment]
the engineer metaclub's ce powercreep plan is proceeding apace, excellent work agent

**Comment by pedr69** [comment]
CE continues to get more stuff, but not many play it...
if the white locker became ablative shielded, and the magboots required an ID to eject, maybe the yellow tide wouldn't just ransack the office roundstart

**Pull request merged**